README.md

```diff
@@ -116,25 +129,25 @@ To enable the language to be selected via the language picker, add an item to th
 `deployment/ansible/group_vars/all`. Setting `rtl` to true will enable right-to-left CSS changes.
 
 
-## Docker
+### Docker
+To update the Docker container images to reflect environment changes (Such as changed Python packages), provision the `app` VM:
+```bash
+vagrant provision app
+```
 
-As a shortcut to building the Docker container images contained within the `app`
-virtual machine, use the `Makefile` at the root of the project:
+### Testing
 
+#### Javascript
+To run the Javascript automated tests, use:
 ```bash
-export DOCKER_HOST=tcp://127.0.0.1:2375
-make {all,app,editor,web}
+./scripts/grunt.sh web test
 ```
 
-This will make use of a Docker client installed on the virtual machine host,
-telling it to communicate with the Docker daemon on the `app` virtual machine.
-
 ## Testing Data
 
 ### Boundaries
 
-To load boundaries, upload the `regions.zip` and `states.zip` files to Ashlar.
-Ashlar runs on localhost:7001. For each file, first upload the file, then select
+To load boundaries, upload the `regions.zip` and `states.zip` files to the schema editor in http://localhost:7000/editor/. For each file, first upload the file, then select
```

Comment:
I was thinking more about this and I think one way to address the Azavea / non-Azavea issue would be to simply acknowledge it: "For developers at Azavea, use the `regions.zip` and `states.zip` files in the DRIVER project folder on the fileshare. For non-Azavea users, upload a zipped shapefile containing the boundaries of the jurisdictions where you plan to operate DRIVER. If you don't have such a shapefile, [Natural Earth](https://www.naturalearthdata.com/features/) is a good place to start."

What do you think about phrasing it that way?

Reply:
If we're OK officially distinguishing between Azavea and non-Azavea I'm good with that 👍

Reply:
We could add that "this project was originally developed by Azavea" as a lead-in, but otherwise I don't see a problem with giving Azavea-specific instructions as long as we provide the best instructions we can for non-Azavea users.